main_test.go

```diff
@@ -713,3 +716,37 @@ func Test_runScorecardSettings(t *testing.T) {
 		})
 	}
 }
+
+func Test_signScorecardResult(t *testing.T) {
+	t.Parallel()
+	// Sign example scorecard results file.
+	scorecardResultsFile := "./testdata/scorecard-results-example.sarif"
+	err := signScorecardResult(scorecardResultsFile)
+	if err != nil {
+		t.Errorf("signScorecardResult() error, %v", err)
+		return
+	}
+
+	// Verify that the signature was created and uploaded to the Rekor tlog by looking up the payload.
+	ctx := context.Background()
+	rekorClient, err := rekor.NewClient("https://rekor.sigstore.dev")
+	if err != nil {
+		t.Errorf("signScorecardResult() error getting Rekor client, %v", err)
+		return
+	}
+	scorecardResultData, err := ioutil.ReadFile("./testdata/scorecard-results-example.sarif")
+	if err != nil {
+		t.Errorf("signScorecardResult() error reading scorecard result file, %v", err)
+		return
+	}
+	uuids, err := cosign.FindTLogEntriesByPayload(ctx, rekorClient, scorecardResultData)
```

Comment:
@asraa is that ok if we upload into the prod TLog each time our tests run? Or is there another dev/test TLog we shoudl use?

Reply:
currently we should use prod/public instance -- there are plans to get a dev tlog, but not yet.